bash.md

```diff
@@ -14,150 +14,119 @@ keywords:
   - Command substitution
 ---
 
-Getting started
----------------
-{: .-three-column}
-
-### Example
-
-```bash
-#!/usr/bin/env bash
 
-NAME="John"
-echo "Hello $NAME!"
-```
-
-### Variables
-
-```bash
-NAME="John"
-echo $NAME
-echo "$NAME"
-echo "${NAME}!"
-```
-
-### String quotes
-
-```bash
-NAME="John"
-echo "Hi $NAME"  #=> Hi John
-echo 'Hi $NAME'  #=> Hi $NAME
-```
-
-### Shell execution
 
-```bash
-echo "I'm in $(pwd)"
-echo "I'm in `pwd`"
-# Same
-```
 
-See [Command substitution](http://wiki.bash-hackers.org/syntax/expansion/cmdsubst)
+Getting started
+---------------
+{: .-five-column}
 
-### Conditional execution
+You should always use a shellcheck (https://github.com/koalaman/shellcheck) if you are writing bash/shell script to avoid unnecesary issues.
 
-```bash
-git commit && git push
-git commit || echo "Commit failed"
-```
+Best practice is to write a POSIX shell script to make the script more functional (working on platforms without dynamic linking) and use bash only if the function is not available on shell (complicated regex, etc..)
```

Comment:
The site also doesn't have (or properly support) paragraphs right after H2's without an H3 heading :(